### DISCUSSION TITLE
Generate PostCSS config on tailwind init

### DISCUSSION THREAD

**Discussion by tylergets** [tailwindlabs/tailwindcss#3607] [Ideas]
Hey! 

Not sure the format here for this, but it would be great if the CLI utility could generate the postcss file alongside the tailwind.config.js

I image something like 
```npx tailwind init --postcss```

Im always thinking of ways to speed up the flow! :)

**Comment by adamwathan** [comment]
Try this ;)

```
npx tailwindcss init -p
```
